### PR TITLE
updates to super-quick start documentation; task #45

### PR DESCRIPTION
**Pull request by jgauglitz**
updates to super-quick start documentation; task #45
added more visuals and updated explanation
Let me know if more edits are needed.

### CODE REVIEW

File: docs/superquickstart.md [hunk 1]
```diff
@@ -1,42 +1,62 @@
...
 
-**IMPORTANT!**: The limitations to this method is that the files uploaded will not reside in your own account (though no one else can see them) and will removed on a monthly basis. Please create your own account for more persistent and reproducible analyses.
+## No log-in required
+
+**IMPORTANT!**: The limitations to this method is that the files uploaded will not reside in your own account (though no one else can see them) and will be removed on a monthly basis. Please create your own account for more persistent and reproducible analyses.
 
 You would be able to use the Super quickstart interface if you have the following requirements:
 
-- [ ] The format of your files is mzXML/mzML/Thermo raw files.
+- [ ] The format of your files is mzXML/mzML (Thermo raw files (.raw)/Bruker raw files (.d) can be converted on GNPS [Checkout this [documentation](fileconversion.md).])
```
Comment (mwang87) [inline docs/superquickstart.md]: This is incorrect. Quickstart does not support Bruker files.

File: docs/superquickstart.md [hunk 2]
```diff
@@ -1,42 +1,62 @@
...
-Once files have been uploaded into appropriate cohorts, enter your email and click "Analyze Uploaded Files with GNPS Molecular Networking".
+Once files have been uploaded into appropriate cohorts, enter your email and click "Analyze Uploaded Files with GNPS Molecular Networking". Entering your email will give you a means of retrieving the job at a later time or if you close the browser.
 
 ![button](img/superquickstart.png)
 
 This will bring you to a GNPS workflow status page. If you entered your email, you will receive a notification once the analysis job is complete. If you did not enter an email, save the URL of the analysis as this will be the only way for you to find the analysis again.
 
+![button](img/job_submission_dotd.PNG.png)
+
+Note: If you drag proprietary files, such as .d files into the Groups, you will be able to start the job, however it will fail, as shown here:
```
Comment (mwang87) [inline docs/superquickstart.md]: No support for .d files.

File: docs/superquickstart.md [hunk 3]
```diff
@@ -45,7 +65,7 @@ Please navigate to the [super quickstart page](https://gnps-quickstart.ucsd.edu/
 
 ### Selecting Files
 
-Simply drag and drop the files out of your feature detection and alignement tool:
+Simply drag and drop the files generated by your feature detection and alignment tool:
```
Comment (mwang87) [inline docs/superquickstart.md]: Add links to FBMN here.

File: docs/superquickstart.md (hunk 1 above)
Comment (jgauglitz) [reply docs/superquickstart.md]: Bruker files can't be directly used, but they can be converted with the fileconversion tool, right? I can remove from here. How do I make changes to the commit?

File: docs/superquickstart.md (hunk 3 above)
Comment (jgauglitz) [reply docs/superquickstart.md]: did in the new commit

File: docs/superquickstart.md (hunk 2 above)
Comment (jgauglitz) [reply docs/superquickstart.md]: removed in the new commit.

File: docs/superquickstart.md (hunk 1 above)
Comment (jgauglitz) [reply docs/superquickstart.md]: I removed from the description in the new commit.

**Pull request merged**